README.md

```diff
@@ -106,9 +106,11 @@ Matrix arguments are NumPy arrays for dense solvers and SciPy Compressed Sparse
 - *I have a squared norm in my cost function, how can I apply a QP solver to my problem?*
   - You can [cast squared norms to QP matrices](https://scaron.info/blog/conversion-from-least-squares-to-quadratic-programming.html) and feed the result to [`solve_qp`](https://qpsolvers.github.io/qpsolvers/quadratic-programming.html#qpsolvers.solve_qp).
 - *I have a non-convex quadratic program. Is there a solver I can use?*
-  - Unfortunately most available QP solvers are designed for convex problems.
-  - If your cost matrix *P* is semi-definite rather than definite, try OSQP.
-  - If your problem has concave components, go for a nonlinear solver such as [IPOPT](https://pypi.org/project/ipopt/) *e.g.* using [CasADi](https://web.casadi.org/).
+  - Unfortunately most available QP solvers are designed for convex problems (i.e. problems for which `P` is positive semidefinite). That's in a way expected, as solving non-convex QP problems is NP hard.
```

Comment:
```suggestion
  - Unfortunately most available QP solvers are designed for convex problems (i.e. problems for which $P$ is positive semi-definite).
```

Solving convex QP problems is [also NP-hard](https://cs.stackexchange.com/a/115549).

Reply:
It's not - [algorithms with polynomial complexity exist since the 80s that solve convex convex QPs](https://link.springer.com/article/10.1007/BF01587086). It's unclear to me what the link you shared says about (continuous as opposed to mixed-integer) convex QPs.

Reply:
Sorry for my cursory feedback on this. You're right, interior point methods exist in polynomial time.

We could even add "... is NP hard, whereas solving convex QPs can be done in polynomial time." here.